client/src/components/Header/constants.ts

```diff
@@ -0,0 +1,20 @@
+import { MenuLink } from './types';
+
+export const MENU_LINKS: MenuLink[] = [
+  {
+    title: 'Home',
+    link: '/',
```

Comment:
We can make this a `.tsx` in the future and put the icons directly in here if we want to.

Reply:
Interesting thought. I might rather have a `icons.tsx` in the case that this boy gets too big. But we'll cross that road when we get there.